### DISCUSSION TITLE
Drivers from latest Session key have null data

### DISCUSSION THREAD

**Discussion by AlessandroCheru** [br-g/openf1#45] [General]
Good evening,
first of all thank you for the API, they are so good!

A report from our side:
When i call API for drivers using latest session key, the data rerurned are partially empty.

https://api.openf1.org/v1/drivers?session_key=latest

Checking with meeting_key=latest, i see some sessions with the complete data, but not the latest one.

Is it possible to have the complete data with the latest session?

Thank you so much!

**Comment by KevinCamacho** [comment]
Also seeing the same, for example, when trying to get the driver information for the race session of the recent Australian and Japanese Grand Prix.

https://api.openf1.org/v1/drivers?session_key=9488
https://api.openf1.org/v1/drivers?session_key=9496

**Comment by br-g** [comment]
Fixed! Thanks for reporting the issue 👍

**Reply by KevinCamacho** [reply]
Validated working - thank you.
